### DISCUSSION TITLE
Multiple Armor Sets

### DISCUSSION THREAD

**Discussion by emeryplyler** [TBoshoven/MinecraftMods#139] [Q&A] [answered]
Hello!
I have somehow stored two armor sets inside the mirror. Are there any rules as to how to store more armor sets or how to take them out? I noticed that it cycles through the armor sets when you equip them from the mirror so this is intended right?

**Comment by TBoshoven** [comment]
Is not designed to hold multiple armor sets.
It's supposed to hold exactly one armor set and swap it with your currently equipped armor when right-clicking.

The armor modifier holds [exactly one replacement armor](https://github.com/TBoshoven/MinecraftMods/blob/1.20.1/MagicMirror/src/main/java/com/tomboshoven/minecraft/magicmirror/blocks/entities/modifiers/ArmorMagicMirrorBlockEntityModifier.java#L57) so I'm really curious about how you managed to store more than one.
Did you manage to attach more than one armor stand perhaps? I think this would be the only explanation, but I'm surprised that would lead to cycling between armors.

Storing more than one armor might be a fun feature though.

**Reply by emeryplyler** [reply]
Ohh I see. I did attach multiple armor stands. I'll take the important armor out just in case. Thank you!

**Reply by TBoshoven** [reply, marked as answer]
Hah, you're also not supposed to be able to attach multiple armor stands :sweat_smile: 
I managed to reproduce this, so I'll write this down as a bug.
I'll also write down support for more than one set of armor as a future enhancement.

At least, due to the way the modifiers are stored on the mirror, you should get back all of the items stored in the mirror when you break it, with no risk of item loss.

**Reply by TBoshoven** [reply]
Issues #140 and #141 for the above.
